examples/server/server.cpp

```diff
@@ -257,10 +258,55 @@ struct llama_server_context
         return true;
     }
 
+    std::vector<llama_token> tokenizePrompt(void)
```

Comment:
Could you make it so it accepts the prompt as argument?
```suggestion
    std::vector<llama_token> tokenizePrompt(json prompt)
```

Reply:
Do you still want the void version overloaded?

Reply:
I don’t know if it’s needed. The reason I ask for this change is that in #2217 I added a second prompt and it would be nice to reuse the code.